ci/receive-hook

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
To maximize system compatibility I'd recommend only depending on `#!/bin/sh` (lots of people have `dash` instead of bash).

Also maybe makes sense to `set -ue` to fail fast if there's anything unexpected.